demo-automation-java/src/test/java/com/example/pages/carForSale/CarForSalePage.java

```diff
@@ -0,0 +1,187 @@
+        return keyword.findElement(carForSales).isEnabled();
+    }
+
+    /*
+     * Return Title Displayed correct or not
+     */
+    public boolean isTitleDisplayedCorrect() {
+        String expectedTitle = "Cars for Sale Nationwide";
+        String actualTitle = keyword.findElement(lblTitle).getText();
+        return actualTitle.equals(expectedTitle);
+    }
+
+    /*
+     * Return Icon Zipcode Displayed or not
+     */
+    public boolean isIconZipcodeDisplayed() {
+        return keyword.findElement(iconZipcode).isDisplayed();
+    }
+
+    /*
+     * Verify the top of the cars for sale section displayed as default
+     */
+    public boolean isTheTopOfTheCarForSalePageSectionDisplayedAsDefault() {
+        if (isSearchTextBoxDisplayed() & isSearchPlaceHolderDisplayedCorrect() & isSearchButtonDisplayed()
+                & isCarForSalesBreadcrumbsEnabled() & isTitleDisplayedCorrect() & isIconZipcodeDisplayed()) {
+            return true;
+        } else
+            return false;
+    }
+
+    /*
+     * Return value list of filter sort displayed correct or not
+     */
+    public void isFilterSortValueListDisplayedCorrect(){
+        Select select = new Select(keyword.findElement(ddlSort));
+        List actualDropdownValues = new ArrayList();
+        for (WebElement element : select.getOptions()) {
+            actualDropdownValues.add(keyword.getText(element));
+        }
+        List expectedDropdownValues = new ArrayList();
+        expectedDropdownValues.add("Relevance");
+        expectedDropdownValues.add("Price - Lowest");
+        expectedDropdownValues.add("Price - Highest");
+        expectedDropdownValues.add("Distance - Closest");
+        expectedDropdownValues.add("Date - Oldest");
+        expectedDropdownValues.add("Date - Newest");
+        expectedDropdownValues.add("Mileage - Lowest");
+        expectedDropdownValues.add("Mileage - Highest");
+        expectedDropdownValues.add("Year - Oldest");
+        expectedDropdownValues.add("Year - Newest");
+        for (int i = 0; i < actualDropdownValues.size(); i++) {
+            Assert.assertTrue(actualDropdownValues.get(i).equals(expectedDropdownValues.get(i)),"The actual result of filter sort doesn't match with expected result");
+        }
+    }
+
+    /*
+     * Return The number of car car item displayed correct as required or not
+     */
+    public boolean isTheNumberOfCarCardItemDisplayedCorrectAsRequired(){
+        List<WebElement> carItem = keyword.findElement(carList).findElements(By.xpath("//div[@data-cmp='inventoryListing']"));
+        if(carItem.size()==25){
+            return true;
+        }
+        else return false;
+    }
+
+    /*
+     * Action click car item
+     */
+    public void clickCarItem(){
+        keyword.click(keyword.findElement(carItem));
+    }
+
+    /*
+     * Return the title of car item
+     */
+    public String getTileOfCarItem(){
+        return keyword.getText(keyword.findElement(lblCarTitle));
+    }
+

```

Comment:
remove redundant space. Should have one space each function

Reply:
Thank you for your comment. I have removed redundant space.